### ISSUE TITLE
Exp Thresholds simplification

### ISSUE THREAD

**Issue by whiteboxsolutions** [jason-allen-oneal/dragon-court#1]
Just noticed that in lib/game.js, we've got expThresholds as an array with a bunch of hardcoded values. It'd probably be cleaner to generate it on the fly based on the base Exp (nextLevelThreshold = currentLevelThreshold * 1.39).

**Comment by jason-allen-oneal** [comment]
I had originally left them hard coded because these were the values I had found while doing initial research. Never really took the time to breakdown the actual formula Fred had used. I do agree it would be more efficient, however.

**Comment by whiteboxsolutions** [comment]
Looks like it's just a 39% increase level by level (after checking about 8 thresholds

Do we have any of the original game code anywhere?

**Comment by jason-allen-oneal** [comment]
No, the original code is nowhere that I can find anymore. I've also lost a lot of my original code in Android from when I started due to a hard drive failure. I was much further along than this lol

**Comment by whiteboxsolutions** [comment]
Ouch - and I take it no chance of Fred providing any assistance in that department?

**Comment by jason-allen-oneal** [comment]
Not so far. He is a lead dev at eBay, I believe, and has yet to answer any of the messages I've sent through various platforms.
